api/tests/integration/infrastructure/repositories/authentication-method-repository_test.js

```diff
@@ -835,6 +835,32 @@ describe('Integration | Repository | AuthenticationMethod', function () {
     });
   });
 
+  describe('#removeAllAuthenticationMethodsByUserId', function () {
+    it('should delete from database all the authentication methods by userId', async function () {
+      // given
+      const userId = databaseBuilder.factory.buildUser().id;
+      databaseBuilder.factory.buildAuthenticationMethod.withGarAsIdentityProvider({
+        id: 123,
+        externalIdentifier: 'externalIdentifier',
+        userId,
+      });
+      databaseBuilder.factory.buildAuthenticationMethod.withPixAsIdentityProviderAndPassword({
+        id: 456,
+        userId,
+      });
```

Comment:
```suggestion
      databaseBuilder.factory.buildAuthenticationMethod.withGarAsIdentityProvider({
        externalIdentifier: 'externalIdentifier',
        userId,
      });
      databaseBuilder.factory.buildAuthenticationMethod.withPixAsIdentityProviderAndPassword({
        userId,
      });
```
L'id est géré coté builder, ce n'est pas nécessaire de les renseigner en dur. 
`id = databaseBuffer.getNextId(),`

Reply:
done